### DISCUSSION TITLE
ergonomy choices of v4.0.0

### DISCUSSION THREAD

**Discussion by harmin-parra** [pytest-dev/pytest-html#710] [General]
Hi

I have noticed some changes between **v3.2.0** and **v4.0.0**, and would like to understand the reason for such changes.

**Number 1**

The `Show all details / Hide all details` link have been positioned to the right.

Why forcing the user to move the mouse all the way to the right to expand or hide ?

**Number 2**

The mouse hover over the first column of the result table, causes the size of the column to change
![v3 2 0](https://github.com/pytest-dev/pytest-html/assets/16239820/ed687a72-729c-4d5a-81f9-b6d3a194f6f9)
![v4 0 0](https://github.com/pytest-dev/pytest-html/assets/16239820/e6333a0b-22c0-424b-ba9f-7a8f791b8c6a)

![vokoscreen-2023-08-09_01-05-36](https://github.com/pytest-dev/pytest-html/assets/16239820/5eca05da-1ac1-457a-bf24-adf71289c6f3)

**Comment by BeyondEvil** [comment]
1. It's one of the steps we're taking to enable a better usage of screen real estate. Another step is the ability to collapse the environment table.
2. That's a side-effect of the show/hide hover. I'd consider it a bug and as such welcome a fix for it.

**Comment by harmin-parra** [comment]
It has been fixed
